CafeTime/CafeTime/AppDelegate.swift

```diff
@@ -8,52 +8,82 @@
 
 import UIKit
 import Firebase
+import SwiftCop
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
+    
     var window: UIWindow?
-
-
+    
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         
+        FIRApp.configure()
+        
         window = UIWindow(frame: UIScreen.main.bounds)
         if let window = window {
-//            let nav = UINavigationController()
-//            nav.pushViewController(LoginVC(), animated: true)
-            
+            window.rootViewController = AuthenticatedUserVC()
             window.backgroundColor = UIColor.white
-            window.rootViewController = LoginAndSignUpVC()
             window.makeKeyAndVisible()
         }
         
-        FIRApp.configure()
+        let saver = GlobalSaver.shared
+        
+        if let email = saver.getUserCredentials().email, let password = saver.getUserCredentials().password  {
+            
+            FIRAuth.auth()?.signIn(withEmail: email, password: password, completion: { (user, error) in
```

Comment:
If we have a AuthManager, probably we should move this call there, and do not call directly FIRAuth.auth()?.signIn, but call AuthManager method